pkg/tuned/tuned.go

```diff
@@ -455,13 +484,17 @@ func (c *Controller) tunedRun() {
 			if c.daemon.reloading {
 				c.daemon.reloading = !profileApplied && !reloadFailed
 				c.daemon.reloaded = !c.daemon.reloading
+				if c.daemon.reloaded {
+					klog.V(2).Infof("profile applied or reload failed, stopping the TuneD watcher")
+					c.tunedTimeout = tunedTimeoutStart // initialize the timeout
+					c.tunedTicker.Stop()               // profile applied or reload failed, stop the TuneD watcher
+				}
+
 				if c.daemon.reloaded {
```

Comment:
I guess these two blocks can be merged

I get lost trying to understand
```
c.daemon.reloading = !profileApplied && !reloadFailed
c.daemon.reloaded = !c.daemon.reloading
if c.daemon.reloaded {
   klog.V(2).Infof("profile applied or reload failed, stopping the TuneD watcher")
```
so I'll trust you with what the comments+log describe!

Reply:
I agree on merging the two blocks.  Will do.  Not sure I agree `reloadFailed` vs. `reloadSuccess` would be easier to follow.  Perhaps my brain works differently in this case. :)